common/wuanalysis/anacommon.cpp

```diff
@@ -46,7 +47,12 @@ void PerformanceIssue::createException(IWorkUnit * wu)
     we->setSeverity(SeverityInformation);
     we->setScope(scope.str());
     we->setPriority((unsigned) statUnits2msecs(cost));
-    we->setExceptionMessage(comment.str());
+    we->setExceptionLineNo(line);
+    we->setExceptionColumn(column);
+    we->setExceptionFileName(filename);
```

Comment:
i think this will set it to blank rather than null if it is not known.  Is that a problem?

Reply:
I don't think it would cause a problem but I'll modify to only set if non-empty string.

Reply:
getProp isn't returning anything (empty string). Could there be a problem with getProp?  Eventually, I modified to use queryProp and it is working.